tests/integration/test_messages_integration.py

```diff
@@ -1437,3 +1437,52 @@ def test_streaming_brief_spinner(capsys, logger, monkeypatch, init_emitter):
         Line("Done stage"),
     ]
     assert_outputs(capsys, emit, expected_err=expected_err, expected_log=expected_log)
+
+
+@pytest.mark.parametrize("output_is_terminal", [True])
+def test_secrets_integrated(capsys, logger, monkeypatch, init_emitter):
```

Comment:
to document the API expectations, it might be worth to also do a "with pause" and print instead of emit.

Reply:
done; I actually found an issue with the test so thanks for pointing it to me!